### DISCUSSION TITLE
Cloudflare Tunnel instead of npm.

### DISCUSSION THREAD

**Discussion by npmaharaj1** [dani-garcia/vaultwarden#3137] [Ideas]
I found that using a Cloudflare tunnel instead of proxy managers such as Nginx is a lot easier and a lot more secure since you don't need to open any ports. Possibly add a segment for this in the README.md file?

Original video tutorial (3rd party) --> https://www.youtube.com/watch?v=hrwoKO7LMzk&t

**Comment by stefan0xC** [comment]
I don't think this should be added to the README (as it is a third party service) but maybe you could add a segment to the [Proxy examples page in the wiki](https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples) that explains how to use this?

**Comment by BlackDex** [comment]
And, everybody can update the wiki :)
